packages/backend/src/database/migrations/20220131130200_migrate_filters.ts

```diff
@@ -117,7 +117,7 @@ export async function down(knex: Knex): Promise<void> {
             FiltersColumnName,
         )
     ) {
-        knex(SavedChartVersionsTableName).update({
+        await knex(SavedChartVersionsTableName).update({
```

Comment:
This is a `down` migration that is missing an `await`. Most likely, it will not complete the down migration quickly, so it's beneficial that we are already identifying some bugs!

Reply:
nice catch